tests/integration/integration_nodejs_test.go

```diff
@@ -1132,6 +1132,26 @@ func TestESMTS(t *testing.T) {
 	})
 }
 
+func TestTSWithPackageJsonInParentDir(t *testing.T) {
+	t.Parallel()
```

Comment:
I believe the reason integration tests aren't marked as parallel elsewhere in the file is because they are automatically marked as parallel in `integration.ProgramTest`